lib/helpers/rule-test-harness.js

```diff
@@ -67,6 +66,12 @@ function getVerifyOptions(template, meta) {
      '<img alt>',
    ]
  *
+ * @param  {Array}  [fix=[]]             - an array of items that describe an input and expected output for a rules --fix support
+ *  [{
+     input: '<button></button>',
+     output: '<button type="button"></button>',
+   }]
```

Comment:
I think I would prefer to couple this with the `bad` list that we already have. if an element in `bad` has an `output` key then we would add tests that run `verifyAndFix()`.

Reply:
Ya, I agree with you now that I’ve done it (and wrote that test plugin). We only attempt to do a fix if you log errors so thing them together makes a lot of sense. When I made the plugin in this PR (in the test suite), I fought with it for a few minutes trying to figure out why it wasn’t running, turns out I was only attempting to mutate and never emitted an error 😩. 

The names there are just a bit off (`template` -> `output`), but I’ll make it work (I might rename `output` to `fixedTemplate` or something).

Reply:
Updated, thanks for pointing this out!